server/app/middlewares/auth.js

```diff
@@ -0,0 +1,24 @@
+import validator from 'email-validator';
```

Comment:
I think the name of this file might be a little misleading as it doesn't actually authenticate but only validates user input